themes/upbound/assets/scss/_color-mode-dark.scss

```diff
@@ -64,6 +64,32 @@
     --search-box-background: var(--body-background);
     --result-path: var(--body-background);
 
+    // API Page
+    --sort-up: url("/img/sort-up-light.svg");
+    --sort-down: url("/img/sort-down-light.svg");
+    --expand-box-background: #{$up-grey-800n};
+    --minus-svg: url("/img/minus-white.svg");
+    --plus-svg: url("/img/plus-150.svg");
```

Comment:
I think this needs to be `img/lightbox/...`, but these are only dark +/-. 

You can get the originals from here:
https://github.com/crossplane/docs/tree/master/themes/geekboot/static/img

Be sure to add the same vars into the `_color-mode-light.scss` file as well.